.travis.yml

```diff
@@ -11,13 +12,12 @@ env:
    - TVOS_SCHEME="DefaultStringConvertible-tvOS"
    - WATCHOS_SCHEME="DefaultStringConvertible-watchOS"
 
-   - IOS_SDK=iphonesimulator9.3
-   - OSX_SDK=macosx10.11
-   - TVOS_SDK=appletvsimulator9.2
-   - WATCHOS_SDK=watchsimulator2.2
+   - IOS_SDK=iphonesimulator10.0
+   - OSX_SDK=macosx10.12
+   - TVOS_SDK=appletvsimulator10.0
+   - WATCHOS_SDK=watchsimulator3.0
 
    matrix:
-   - DESTINATION="OS=8.1,name=iPhone 4s"      SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" POD_LINT="YES"
```

Comment:
@jessesquires indeed... I remember Travis didn't support iOS 8 on Xcode 8 a couple of moths ago. It seems they fixed it.